docs/index.md

```diff
@@ -0,0 +1,58 @@
+A guide to [TBD](https://github.com/rd2/tbd "TBD source code repository on GitHub") - an [OpenStudio Measure](https://nrel.github.io/OpenStudio-user-documentation/reference/measure_writing_guide/ "A guide to writing OpenStudio Measures") that auto-detects __major__ thermal bridges (like balconies, parapets and corners) and then __derates__ outside-facing opaque constructions (walls, roofs and exposed floors). The guide is mainly written for architects, technologists and envelope consultants who are new to energy simulation, and to OpenStudio in particular.
```

Comment:
@macumber: Keeping the /docs solution for the moment (may still want to try switching from gh-pages soonish). But likely best solution is to remove /docs, as to avoid confusing others.